core/google/cloud/exceptions.py

```diff
@@ -250,3 +259,44 @@ def _walk_subclasses(klass):
     code = getattr(_eklass, 'code', None)
     if code is not None:
         _HTTP_CODE_TO_EXCEPTION[code] = _eklass
+
+_GRPC_ERROR_MAPPING = {
```

Comment:
1. You want two empty lines between top-level definitions
1. This will fail with a `NameError` if `StatusCode` fails import